how-to-install-ruby.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+permalink: /how-to-install-ruby/
+---
+
+# Installing Ruby using rbenv
+
+## Install dependencies
+
+~~~
+sudo apt-get update
+~~~
+~~~
+sudo apt-get install git-core curl zlib1g-dev build-essential libssl-dev libreadline-dev libyaml-dev libsqlite3-dev sqlite3 libxml2-dev libxslt1-dev libcurl4-openssl-dev python-software-properties libffi-dev
+~~~
+
+## Get rbenv
+~~~
+cd
+git clone git://github.com/sstephenson/rbenv.git .rbenv
+echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
```

Comment:
My instructions are based on [this tutorial from DigitalOcean](https://www.digitalocean.com/community/tutorials/how-to-install-ruby-on-rails-with-rbenv-on-ubuntu-14-04#install-rbenv). See the note in yellow.

I've searched the web ([1](http://unix.stackexchange.com/questions/26047/how-to-correctly-add-a-path-to-path), [2](http://superuser.com/questions/183870/difference-between-bashrc-and-bash-profile/183980#183980)) and discussed with Clément, and we came to the conclusion that `.bashrc` will be a good option for newbies, and that those who know what they're doing will use `.profile` or `.bash_profile` if they prefer.